src/renderer/bootstrap.tsx

```diff
@@ -124,7 +122,9 @@ export async function bootstrap(comp: () => Promise<AppComponent>, di: Dependenc
   initializers.initCatalogCategoryRegistryEntries();
 
   logger.info(`${logPrefix} initializing Catalog`);
-  initializers.initCatalog();
+  initializers.initCatalog({
```

Comment:
Notice: manual initialization of stuff here and elsewhere is something that can be done better. Soon.